### DISCUSSION TITLE
"Something went wrong"

### DISCUSSION THREAD

**Discussion by nightznero** [community/community#112208] [Mobile]
### Select Topic Area

Bug

### Body

`Something went wrong` 
That's all i see since very end 2023,early 2024.
![Screenshot_20240312_072710_GitHub](https://github.com/community/community/assets/61750641/8e9bedfc-b533-49ce-9659-f8a55990eb2e)
Gloom (GitHub client from MateriiApps) also doesn't work.
A 13 - OneUI 5.1
v1.152.0

**Comment by kangarruu** [comment]
@nightznero We're sorry you're experiencing this! Thanks for writing in ❤️  Can you elaborate on the workflow that got you to this point? You've been seeing this error consistently after upgrading several versions from the Play Store including v1.152.0? Is this with a working network connection? Any additional details would be really valuable to help get this working for you again.

**Reply by nightznero** [reply]
Its not internet connection. The issues started just from nowhere. I didn't change anything in my workflow that could make the issue. I created with site feed (https://github.com/orgs/community/discussions/81651) since then also app stopped working properly. Tested on VPN,wifi,lte, results are the same, one day GitHub just stopped working fir me

**Reply by kangarruu** [reply]
Thanks again for bringing this to our attention. Keep an eye out for a fix in the next release!

**Reply by francoiscoetzee85** [reply]
I just updated to the lates and now seeing the same error, I'm not able to view any code it simply just tells me something when wrong. 

Can verify everything was working before I updated.

**Reply by nightznero** [reply]
for me its still unusable so huh...welcome in club and start using a web bcs its not gonna be fixed soon

**Reply by kangarruu** [reply]
Sorry to hear that 🙁 Are you seeing this error only on the Home tab? Can you navigate elsewhere in the app using the other navigation tabs? 

Could you please verify that you're still able to access these resources in the browser, and if so, would you be willing to check if you're able to open links to files or repositories if you click on a direct link?

**Reply by francoiscoetzee85** [reply]
The error occurs on every code repo I try to view and yes the web browser works for the same repositories.

**Reply by francoiscoetzee85** [reply]
Just to elaborate, I can navigate to repositories roots but clicking into them produces the error

**Reply by francoiscoetzee85** [reply]
Found a fix Navigate to Profile (In the bottom-right) >
Settings (Cog icon button) > Manage Accounts >
Edit > Delete (on the left) > Sign out (on the right)
• A popup opened saying that "We couldn't sign out you successfully because some token was not found" and options to OK (which is sort of Cancel) and "Sign out anyway".
• I said OK once without paying attention carefully and this time I choose "Sign out anyway"
• Signed in again to see that I could access
Repositories and Pull Requests.

**Reply by supermarin** [reply]
Same thing over here and it's been happening for a while

**Reply by nightznero** [reply]
@kangarruu , Navigate button works but navigate to others pages still shows the same error,links from browser to app makes same results. On Browser i can access normally as it should,on browser only home page doesn't work cuz its shows `Update to your homepage feed` and `thats all`. No events,annoucments,relases or sponsors shows up. That 2things is broken since end 2023

**Comment by senior-software-engineers** [comment]
Profile > Settings > Manage Accounts > Edit > Delete > Sign out and then signing back in, which allowed access to repositories and pull requests

**Comment by mtgralanbr** [comment]
> Profile > Settings > Manage Accounts > Edit > Delete > Sign out and then signing back in, which allowed access to repositories and pull requests
